### PR TITLE
add softmax layer

### PR DESCRIPTION
**Pull request by kayzliu**
Changes:
1. modified sigmoid to softmax
2. simplified matrix multiplication

### CODE REVIEW

File: model.py [hunk 1]
```diff
@@ -26,19 +26,19 @@ def __init__(self, num_classes, inter1, lambda_1):
...
+		self.weight = nn.Parameter(torch.FloatTensor(inter1.embed_dim, num_classes))
 		init.xavier_uniform_(self.weight)
 		self.lambda_1 = lambda_1
 
 	def forward(self, nodes, labels, train_flag=True):
 		embeds1, label_scores = self.inter1(nodes, labels, train_flag)
-		scores = self.weight.mm(embeds1)
-		return scores.t(), label_scores
+		scores = torch.mm(embeds1, self.weight)
+		return scores, label_scores
```
Comment (YingtongDou) [inline model.py]: Add `sigmoid` function before returning them, it will make the `gnn_logits`, `label_logits` variable names meaningful.

**Review by YingtongDou — CHANGES_REQUESTED** [review]

File: model.py (hunk 1 above)
Comment (kayzliu) [reply model.py]: Modified as per the requirement.

**Review by YingtongDou — APPROVED** [review]
